src/pages/FoodShow.js

```diff
@@ -2,12 +2,19 @@ import React, { Component } from 'react';
 
 class FoodShow extends Component {
     render() {
+      console.log(this.props.food);
         return (
             <div>
-                <h2> Food Show</h2>
+                <h2>Food Show</h2>
+                {this.props.food && <div>
+                <p> {this.props.food.dishname}</p>
+                <p> {this.props.food.genre}</p>
+                <p> {this.props.food.category}</p>
+                <img src={this.props.food.image} width="200" height="200" />
```

Comment:
If you have images that are not squares you may find giving a height and a width will distort your image.

Reply:
Thanks for the input!